app/controllers/application_controller.rb

```diff
@@ -1,12 +1,11 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
 
-  before_action :intercept_html_requests
```

Comment:
Originally done to allow Angular Router to take over.  Does something now replace this code, or have things changed with rails 4?